### PR TITLE
Add postgres_fdw to database

### LINKED ISSUE #2772: Add Supawrapers and postgres_fdw to Postgres docker image
Add supabase wrapper -> https://supabase.com/docs/guides/database/extensions/wrappers/overview
Add postgres_fwd -> https://docs.postgresql.fr/9.6/postgres-fdw.html 

> Old 
>*For internal team* 
>
> Add https://github.com/pgspider/sqlite_fdw to Postgres production docker image
> This will be useful to access remote objects for https://github.com/openstatusHQ/openstatus which are willing to do a > > POC with us

### PR DESCRIPTION
**Pull request by martmull**
close #2772
- add `postgres_fdw` extension to database
- add `supa_wrappers` extensions to database

### CODE REVIEW

File: infra/build/postgres/Dockerfile [hunk 1]
```diff
@@ -23,16 +23,22 @@ RUN set -eux; \
...
 RUN dpkg --install pg_graphql.deb
-RUN cp /usr/share/postgresql/15/extension/pg_graphql* /opt/bitnami/postgresql/share/extension/
-RUN cp /usr/lib/postgresql/15/lib/pg_graphql* /opt/bitnami/postgresql/lib/
+RUN cp /usr/share/postgresql/${PG_GRAPHQL_VERSION}/extension/pg_graphql* /opt/bitnami/postgresql/share/extension/
+RUN cp /usr/lib/postgresql/${PG_GRAPHQL_VERSION}/lib/pg_graphql* /opt/bitnami/postgresql/lib/
+
+# Install precompiled supabase wrappers extensions
+RUN curl -L "https://github.com/supabase/wrappers/releases/download/v${WRAPPERS_VERSION}/wrappers-v${WRAPPERS_VERSION}-pg${PG_MAIN_VERSION}-${TARGETARCH}-linux-gnu.deb" -o wrappers.deb
+RUN dpkg --install wrappers.deb
+RUN cp /usr/share/postgresql/${PG_GRAPHQL_VERSION}/extension/wrappers* /opt/bitnami/postgresql/share/extension/
```
Comment (charlesBochet) [inline infra/build/postgres/Dockerfile:37]: I'm surprised about PG_GRAPHQL_VERSION being used for wrappers?

**Review by charlesBochet — COMMENTED** [review]
@martmull Left one comment!

File: infra/build/postgres/Dockerfile (hunk 1 above)
Comment (martmull) [reply infra/build/postgres/Dockerfile:37]: yes, it is a mistake thanks. Even for `pg_graphql` is it weird isn't it?

**Review by charlesBochet — APPROVED** [review]
LGTM!

**Pull request merged**